lib/repositories/firebase/consent_repository.dart

```diff
@@ -1,11 +1,47 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:get/get.dart';
 import 'package:pispapp/models/consent.dart';
 import 'package:pispapp/repositories/interfaces/i_consent_repository.dart';
 
-class ConsentRepository implements IConsentRepository {
+class ConsentRepository extends GetxService implements IConsentRepository {
```

Comment:
Why?

Reply:
"GetxService is not automatically disposed. It is ideal for situations where, once started, that service will remain in memory"

I want to be able to keep the ConsentRepository in memory so that the list of accounts (which could be used in a lot of places) can be cached in memory